websites/R/Roblox/presence.ts

```diff
@@ -33,14 +36,18 @@ presence.on("UpdateData", async () => {
   ) {
     profileName = <HTMLHeadingElement>(
       document.querySelector(
-        "div.profile-header-top > div.header-caption > div.header-title > h2"
+        "div.profile-header-top > div.header-caption div.header-title > h2"
```

Comment:
you sure this is correct?

Reply:
Yeah, it worked for me. There was an extra div in between that was preventing this.